src/App/ConsolePage.xaml.cs

```diff
@@ -520,7 +507,5 @@ public void Dispose()
         private int _cmdHistoryIndex;
         private string _containerIp;
         private const int MaxCmdHistory = 100;
-        private const int MaxBlocks = 10; // @500 lines per block this is 5000 lines or 10 commands maximum
-        private const int MaxLinesPerBlock = 500;

```

Comment:
Is a code-defined max size no longer necessary now that we're using a textbox? How much data can a textbox handle?

Reply:
It was to store 10 command outputs before. I am currently only showing output for 1 command.